javascript-carousel/main.js

```diff
@@ -0,0 +1,76 @@
+var $pokemon = document.querySelector('.pokemon');
+var $dots = document.querySelectorAll('.dot');
+var $container = document.querySelector('.container');
+var $rightButton = document.querySelector('.button-right');
+var $leftButton = document.querySelector('.button-left');
+var clicks = 0;
+
+var images = [
+  {
+    src: 'images/001.png',
+    alt: '001'
+  },
+  {
+    src: 'images/004.png',
+    alt: '004'
+  },
+  {
+    src: 'images/007.png',
+    alt: '007'
+  },
+  {
+    src: 'images/025.png',
+    alt: '025'
+  },
+  {
+    src: 'images/039.png',
+    alt: '039'
+  }
+];
+
+var interval = setInterval(startLoop, 3000);
+
+function startLoop() {
+  clicks++;
+  if (clicks > 4) {
+    clicks = 0;
+  }
```

Comment:
Instead of using a magic hard-coded number here, you should make use of the `length` of the array that contains the values you want to loop over.